com.github.Flacon.appdata.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The file should be upstreamed and installed by project's build system.

Reply:
I've added it as a feature request:
https://github.com/flacon/flacon/issues/101

I've also added a note to the file

Reply:
Issue resolved upstream. Should be included with v5.4.1